wallet/src/de/schildbach/wallet/ui/buy_sell/BuyAndSellIntegrationsActivity.kt

```diff
@@ -223,7 +198,6 @@ class BuyAndSellIntegrationsActivity : LockScreenActivity() {
                                 balance
                             )
                         }
-                        loadingDialog?.dismiss()
```

Comment:
This causes a crash since `Success` is reached before the activity is created. Removing this progress dialog solves the problem. Other cases of dismissing a progress dialog are called in a LiveData observer which should respect the lifecycle automatically.